Next version

hansvancalster opened the pull request:
development branch for next version of the package

falkmielke committed:
silence_unpack_mht: suppress output of read_lines

hansvancalster committed:
add Rstudio project id

hansvancalster committed:
bump to 0.0.4

hansvancalster committed:
fix indentation linters

hansvancalster committed:
Merge branch 'next-version' into silence_unpack_mht

falkmielke committed:
silence_unpack_mht: explicit packages; ows4R

falkmielke committed:
silence_unpack_mht: two lintr comments

falkmielke committed:
silence_unpack_mht: reticulate not found

falkmielke committed:
silence_unpack_mht: url warning (checklist)

falkmielke committed:
silence_unpack_mht: typo in URL

hansvancalster committed:
update checklist.yml

falkmielke committed:
silence_unpack_mht: add news item

falkmielke committed:
silence_unpack_mht: (merge local changes)

falkmielke committed:
silence_unpack_mht: too long news

(checklist complaint -> linebreaks inserted)

falkmielke committed:
silence_unpack_mht: correct spelling "line break"

hansvancalster committed:
Merge pull request #19 from inbo/silence_unpack_mht

silence `unpack_mht`: suppress output of read_lines

hansvancalster committed:
Match text description with code example

hansvancalster committed:
update gitignore

hansvancalster committed:
organisation yml

hansvancalster committed:
update gha

hansvancalster committed:
update pkgdown

hansvancalster committed:
update description

hansvancalster committed:
update citation

hansvancalster committed:
add ror

hansvancalster committed:
add function get_wcs_layers

hansvancalster committed:
update documentation

hansvancalster committed:
update vignette

hansvancalster committed:
update documentation

hansvancalster committed:
linter fix

hansvancalster committed:
update NEWS

ThierryO commented on R/crs.R:
```diff
@@ -47,8 +47,12 @@ scalefactor_lcc <- function(par_deg, par1_deg, par2_deg) {
 #'
 #' @keywords internal
```
Use  the `noRd` tag when documenting internal function. This suppresses the building of the help files but still allows to document in the same manner as the exported functions.
```suggestion
#' @keywords internal
#' @noRd
```

hansvancalster committed:
Update R/crs.R

Co-authored-by: Thierry Onkelinx <ThierryO@users.noreply.github.com>

hansvancalster committed:
add noRd tag for internal functions

hansvancalster committed:
add missing plot method

hansvancalster committed:
change VignetteIndexEntry

hansvancalster committed:
lower case dhmv in url

hansvancalster committed:
update checklist.yml

hansvancalster committed:
use cranhaven as fallback for recently archived CRAN packages

hansvancalster committed:
update checklist

hansvancalster committed:
move cranhaven to correct place

ThierryO commented on .github/workflows/check_on_different_r_os.yml:
```diff
@@ -37,15 +37,14 @@ jobs:
         with:
           r-version: ${{ matrix.config.r }}
-          extra-repositories: https://inbo.r-universe.dev
+          extra-repositories: https://inbo.r-universe.dev, https://cranhaven.r-universe.dev
```
I think you need to drop the comma. Just use spaces to separate the list of repositories.

hansvancalster committed:
syntax suggestion

hansvancalster committed:
add mercatornet WCS service

hansvancalster committed:
demonstrate new WCS layers

hansvancalster committed:
add new feature item

hansvancalster committed:
spelling

hansvancalster committed:
add new get_feature_ogc function

hansvancalster committed:
update NAMESPACE

hansvancalster committed:
update Rd

hansvancalster committed:
update NEWS

hansvancalster committed:
update documentation

hansvancalster committed:
fix pagination bug

hansvancalster committed:
smaller bbox

hansvancalster committed:
robust version of unpack_mht

hansvancalster committed:
update NEWS

hansvancalster committed:
use ... differently

hansvancalster committed:
update documentation

hansvancalster committed:
update vignette

hansvancalster committed:
update gitignore and Rbuildignore

hansvancalster committed:
linters

hansvancalster committed:
add word to dic

hansvancalster committed:
reduce cyclomatic complexity

hansvancalster committed:
linter

hansvancalster committed:
spelling

hansvancalster committed:
add @family

hansvancalster committed:
update documentation

hansvancalster committed:
switch from httr to httr2

hansvancalster committed:
move get_feature_wfs examples to the back

hansvancalster committed:
bump to v0.1.0

hansvancalster committed:
update batches

hansvancalster commented:
@falkmielke This PR contains some changes to your vignette and the unpack_mht() function. Can your review these changes and if you can spare the time, feel free to check out the other new features as wel (mainly get_feature_ogc() and a new WCS mercatornet, which has many many raster layers available). I also added functions to get the list of available layers from either WFS, WCS or OGC API features services.

hansvancalster committed:
bump version in NEWS

hansvancalster commented:
Unit tests are not yet added, but is not something I want to include in this PR. The wfs_wcs.Rmd vignette is probably a good start to explore the new functions.

falkmielke committed:
fm_review: add namespaces and requirements

falkmielke committed:
fm_review: more namespace and `require_pkgs` usage

falkmielke committed:
fm_reviews: separate file get_wcs_url(*)

There was a cross dependency:
- get_coverage_wcs would call get_wcs_layers...
- but get_wcs_layers depended on get_wcs_url from get_coverage_wcs

I am certain R can handle this situation, but thought it is good to
isolate the URL handler for clarity.

falkmielke reviewed (approved):
I have checked the main code files, and will add some formal suggestions in a separate PR.
However, those are just formal, for code dependency robustness and dev reading convenience. The PR seems fine as is.
Kudos for the new OGC feature. The function works in line with the previous features, I could test the example, and particularly like the use of sub-functions and the outstanding documentation. Thank you for the work on this!

(The only thing I miss is a tutorial or vignette with more context about the OGC standard and use of this function. However, lack of a vignette should not keep people from using it right now.)

hansvancalster committed:
Merge pull request #23 from inbo/fm_reviews

minor/code-formal revisions and suggestions for the `next-version`

hansvancalster committed:
removed require_pkgs() for Imported packages

hansvancalster committed:
fix spelling issues and update documentation

hansvancalster commented:
> I have checked the main code files, and will add some formal suggestions in a separate PR. However, those are just formal, for code dependency robustness and dev reading convenience. The PR seems fine as is. Kudos for the new OGC feature. The function works in line with the previous features, I could test the example, and particularly like the use of sub-functions and the outstanding documentation. Thank you for the work on this!

Thanks!
 
> (The only thing I miss is a tutorial or vignette with more context about the OGC standard and use of this function. However, lack of a vignette should not keep people from using it right now.)

It is in the `wfs_wcs.Rmd` vignette. The context is written in the introduction and all examples now also demonstrate OGC API features (when available).

hansvancalster committed:
replace invalid URL by new CQL2 specs

hansvancalster committed:
ignore docs folder

hansvancalster committed:
fix allowed note
